setup.py

```diff
@@ -10,10 +10,18 @@
 
 exec(compile(version_file_contents, version_file_name, "exec"), ver_dic)
 
+try:
+    import mirgecom  # noqa: F401
+except ModuleNotFoundError:
+    raise RuntimeError("Install MIRGE-Com as:\n"
+                       " - git clone https://github.com/illinois-ceesd/emirge\n"
+                       " - cd emirge\n"
+                       " - ./install.sh")
+
```

Comment:
Not quite true i.e. we could just make sure the correct versions of loopy/pytato are installed. Maybe think about moving this to some other place and relaxing the constraints here.

Reply:
Dropped for now.